src/comm/src/switchboard.rs

```diff
@@ -70,6 +70,8 @@ where
     rendezvous_table: Mutex<router::RoutingTable<u64, C>>,
     /// Task executor, so that background work can be spawned.
     executor: tokio::runtime::Handle,
+    /// Maximum length of encoded messages (1 GiB by default)
```

Comment:
I don't think this comment is quite right. 1GB is only the default if you use `Switchboard::local()`, though, which is only intended for tests.

I think it would actually be nicer if we made it true, though! I.e., if `Switchboard::new` installed a maximum frame size of 1GB, and then the one caller (materialized/lib.rs) that wanted to adjust it could do so via a `set_max_frame_length` call.